### PR TITLE
Improve Contributor Experience

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- Checking the patch commit message function as well.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,3 +25,8 @@ Currently the following reference material is pulled from other repos:
 - [Python SDK](https://www.balena.io/docs/reference/sdk/python-sdk/) sourced from https://github.com/balena-io/balena-sdk-python/blob/master/DOCUMENTATION.md
 
 Version numbers and commit messages are automatically added to the `CHANGELOG.md` file by the CI build flow, after a pull request is merged. It should not be manually edited.
+
+asdasd
+blah
```
Comment (stathismor) [inline CONTRIBUTING.md:30]: blah blah? :wink:

Comment (vipulgupta2048) [reply CONTRIBUTING.md:30]: @stathismor No worries, I was testing the methods one can add `Change-type` on their commits to inform versionbot of the change-type for semantic versioning. Re:  `versionbot-pr-draft` label, it makes sure the PR can't be auto-merged by the balena-bulldozer until the label is present on a PR. Resolving. 

cc: [Discussion Thread](https://www.flowdock.com/app/rulemotion/r-beginners/threads/LDHkRR-FUcDF04VCtRkNWUvc-Zc)

**Review by stathismor — DISMISSED** [review]
Just a small comment about the second screenshot. Wondering if it should:
1. Have the PR subject in imperative form (Edits -> Edit)
2. Contain some text in the subject, to demonstrate and encourage providing context, as the first screenshot does.

**Comment by vipulgupta2048** [comment]
@stathismor took that screenshot straight from the release party actually. With Alex M's comments, it going to go anyways. Commit coming right up.,

**Comment by vipulgupta2048** [comment]
Thanks Versionbot, want more people's eyes on this before merging.
cc: @pdcastro when you have time.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,24 +1,104 @@
...
+
+- `patch` - For tiny changes that include typo fixes, updating screenshots, adding clarification notes, etc.
+- `minor` - For moderate changes that include updating packages, rewriting documentation, adding new features.
+- `major`- For substantial changes that include updating large parts of the documentation or migrating to new frameworks. 
+
+### Creating Commits using the GitHub UI
+
+For folks using the Github UI to create new pull requests, check out the following guides in order to get started. 
+
+1. [Editing files and proposing changes](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository) on a GitHub Repository. When committing changes, here's how you can add a `Change-type` as a prefix to the commit. Adding `patch:` before your commit will mark the change-type as `patch` successfully.
```
Comment (pipex) [inline CONTRIBUTING.md:19]: Is it just before the commit message or it can work with the PR message as well? What happens when a PR has multiple commits?

Comment (vipulgupta2048) [reply CONTRIBUTING.md:19]: Thanks @pipex for your 👀 on this. 

1. Yes, just before "at least" one commit message would do. No need to sign commits as well. I have added a screenshot here + command-line way too. 
1.5. No, it won't work as a PR message, if you haven't yet created a PR on GitHub (like before hitting the "Create Pull Request" button) you can still go back to the previous page and add it. I have done it several times today. 

2. Again, until and unless you have it on at least on one commit. It's good. 

Anything from what I said above, should I add it as a clarification in the document.

Comment (vipulgupta2048) [reply CONTRIBUTING.md:19]: Added it with some bold.

**Review by pdcastro — APPROVED** [review]
LGTM 👍

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci retest

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Pull request merged**